src/promptflow-evals/promptflow/evals/evaluators/qa/__init__.py

```diff
@@ -0,0 +1,46 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+__path__ = __import__("pkgutil").extend_path(__path__, __name__)  # type: ignore
+
+from promptflow import load_flow
+from promptflow.entities import AzureOpenAIConnection
+from pathlib import Path
+
+
+def init(model_config: AzureOpenAIConnection):
```

Comment:
Will qa evaluator include safety metrics as well? if so, then we will need project_scope and credential as inputs as well

Reply:
For this kind of composite evaluator, do we want user to pick what metrics to run? or we just run all evaluators under it?

Reply:
I would prefer content safety evaluators to be a separate one since they all are very similar. And customer can add it to evaluator list is needed.

Reply:
I did think of it. Since this was first draft did not add it. We can provide configuration on which evaluators to pick.